### DISCUSSION TITLE
How to get installed PSWSMan 3.0.0

### DISCUSSION THREAD

**Discussion by TripetWorkEma** [jborean93/PSWSMan#12] [Q&A]
Dear Jordan,

We've tried to install PSWSMan
Install-Module -Name PSWSMan -Scope CurrentUser

as described here: https://github.com/jborean93/PSWSMan/tree/main

the issue is that it installs the older version 2.3.1
how can we install the newest unreleased version 3.0.0?

Thank you, 
Best regards
David

**Comment by jborean93** [comment]
It's not currently in the PSGallery as the use of the hooking mechanism doesn't work on arm64 based hosts. The only way to use this v3 build is to download the source and build it manually right now.
